bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/TestPutGetDelete.java

```diff
@@ -64,4 +69,129 @@ public void testPutGetDelete() throws IOException {
     Assert.assertFalse(table.exists(get));
     table.close();
   }
+
+  @Test
+  public void testGetWithNullInputs() throws IOException {
+    Table table = getDefaultTable();
+    byte[] rowKey = dataHelper.randomData("rowKey-");
+    byte[] qualifier = dataHelper.randomData("qualifier-");
+    byte[] value = dataHelper.randomData("test-data");
+
+    table.put(new Put(rowKey).addColumn(COLUMN_FAMILY, qualifier, value));
+
+    Exception actualError = null;
+
+    try {
+      table.get((Get) null);
+    } catch (Exception ex) {
+      actualError = ex;
+    }
+    assertNotNull(actualError);
+    actualError = null;
+
+    Get get = new Get(rowKey).addFamily(Bytes.toBytes(""));
+    try {
+      table.get(get);
+    } catch (Exception ex) {
+      actualError = ex;
+      // TODO(rahulkql): Verify if this is bcoz of family not found.
+      System.out.println("Verify if this is bcoz of family not found.");
+      ex.printStackTrace();
```

Comment:
is it intended ?

Reply:
Thanks for review!
It was for test purpose, I forgot to remove those